app/models/pydantic/creation_options.py

```diff
@@ -240,7 +248,8 @@ class PixETLCreationOptions(RasterTileSetAssetCreationOptions):
     source_uri: Optional[List[str]] = Field(
         description="List of input sources. Sources must be the URI of either a "
         "tiles.geojson file on S3 or a folder (prefix) on S3 or GCS. "
-        "Features in tiles.geojson must have path starting with either /vsis3/ or /vsigs/",
+        "Features in tiles.geojson must have path starting with either /vsis3/ or /vsigs/"
+        "auxiliary_assets is ignored if source_uri is set (for creating new versions)",
```

Comment:
It would of course be better to raise an error immediately, rather than just ignoring it. I know that's not in scope for this PR, but since you're already adding the doc could you add a TODO to enforce this?